html5.html

```diff
@@ -1 +1 @@
-<!DOCTYPE html>
\ No newline at end of file
+<!doctype html>
```

Comment:
You can also [remove the space](https://twitter.com/mathias/status/1048098677703753729).

```suggestion
<!doctypehtml>
```

Reply:
The whitespace is required

Source:

https://html.spec.whatwg.org/multipage/syntax.html#the-doctype

A DOCTYPE must consist of the following components, in this order:

1. A string that is an ASCII case-insensitive match for the string "<!DOCTYPE".
2. **One or more ASCII whitespace.** ⭐ 
3. A string that is an ASCII case-insensitive match for the string "html".
4. Optionally, a DOCTYPE legacy string.
5. Zero or more ASCII whitespace.
6. A U+003E GREATER-THAN SIGN character (>).